### PR TITLE
[release/3.1.3xx] Revert to 3.1.2 runtime

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Review by sfoslund — APPROVED** [review]

**Review by wtgodbe — CHANGES_REQUESTED** [review]
Should pin HostModel here as well

**Comment by mmitche** [comment]
> Should pin HostModel here as well

Are we sure it matters here?

**Comment by wtgodbe** [comment]
> Are we sure it matters here?

No, but I don't see the downside given the logic we applied to pinning it in dotnet/sdk

**Comment by wtgodbe** [comment]
@mmitche this needed the alpine3.6 image update also

**Review by wtgodbe — APPROVED** [review]

**Comment by wtgodbe** [comment]
Not sure why this is unable to fetch the alpine3.6 image when it was able to fetch the older one previously - I grabbed it from the list in the ReadMe of https://github.com/dotnet/dotnet-buildtools-prereqs-docker. @MattGal @mthalman @MichaelSimons any idea why the build can't use `microsoft/dotnet-buildtools-prereqs:alpine-3.6-3e800f1-20190501005341` ?

**Comment by MattGal** [comment]
> Not sure why this is unable to fetch the alpine3.6 image when it was able to fetch the older one previously - I grabbed it from the list in the ReadMe of https://github.com/dotnet/dotnet-buildtools-prereqs-docker. @MattGal @mthalman @MichaelSimons any idea why the build can't use `microsoft/dotnet-buildtools-prereqs:alpine-3.6-3e800f1-20190501005341` ?

"Works on my box".  Perhaps you're not fully specifying the registry (and thus getting it from dockerhub and not mcr?)

image

**Comment by MattGal** [comment]
Also note that Alpine 3.6 is EOL so base images might be missing; usually docker stuff is never really removed though.

File: scripts/docker/alpine.3.6/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 
 # Dockerfile that creates a container suitable to build dotnet-cli
-FROM microsoft/dotnet-buildtools-prereqs:alpine-3.6-3148f11-20171119021156
+FROM microsoft/dotnet-buildtools-prereqs:alpine-3.6-3e800f1-20190501005341
```
Comment (MichaelSimons) [inline scripts/docker/alpine.3.6/Dockerfile]: This is using the wrong repo.  `microsoft/dotnet-buildtools-prereqs` has been out of support for a long time now although all images that were published there remain there.  You will need to update the `mcr.microsoft.com/dotnet-buildtools/prereqs` repository and everything should work.

Comment (wtgodbe) [reply scripts/docker/alpine.3.6/Dockerfile]: Thanks!

**Pull request merged**